src/pudl/metadata/dfs.py

```diff
@@ -231,3 +233,135 @@
 From FERC Form 1 page 219, Accumulated Provision for Depreciation of electric
 utility plant (Account 108).
 """
+
+EIA_SECTOR_AGGREGATE_ASSOCIATION = pd.read_csv(
+    StringIO(
+        """
+sector_agg,sector_id_eia
+electric_utility,1
+ipp_non_cogen,2
+ipp_cogen,3
+commercial_non_cogen,4
+commercial_cogen,5
+industrial_non_cogen,6
+industrial_cogen,7
+all_sectors,1
+all_sectors,2
+all_sectors,3
+all_sectors,4
+all_sectors,5
+all_sectors,6
+all_sectors,7
+all_ipp,2
+all_ipp,3
+all_commercial,4
+all_commercial,5
+all_industrial,6
+all_industrial,7
+electric_power,1
+electric_power,2
+electric_power,3
+"""
+    ),
+)
+"""Association table describing the many-to-many relationships between plant sectors
+and various aggregates in fuel_receipts_costs_aggs_eia.
+"""
+
+# PEL is defined as "Summation of all petroleum liquids (distallte fuel oil, jet fuel,
+# residual fuel oil, kerosense waste oil and other petroleum liquids)"
+# Missing from this list are all the "other" categories of gases: OG, BFG, SGP, SC, PG
+# Those gases combine for about the same total MMBTU as DFO, about 0.2% of all reported
+# fuel receipts
+EIA_FUEL_AGGREGATE_ASSOCIATION = pd.read_csv(
+    StringIO(
+        """
+fuel_agg,energy_source_code_eia
+bituminous_coal,BIT
+sub_bituminous_coal,SUB
+lignite_coal,LIG
+all_coal,BIT
+all_coal,SUB
+all_coal,LIG
+all_coal,WC
+natural_gas,NG
+petroleum_coke,PC
+petroleum_liquids,DFO
+petroleum_liquids,RFO
+petroleum_liquids,JF
+petroleum_liquids,KER
+petroleum_liquids,WO
+    """
+    ),
+)
+"""Association table describing the many-to-many relationships between fuel types
+and various aggregates in fuel_receipts_costs_aggs_eia."""
+
+
+# This is NOT an association table! The bulk data has a region 'USA' that aggregates
+# all states (not sure about territories)
+STATES = pd.read_csv(
```

Comment:
I made the table this way after searching for "arkansas" and finding only two other tables. They had name, abbreviation and FIPS. So this seems sufficient to replace our current usage of that info. I don't know what other geographies we work with in other parts of PUDL, but they don't seem to be combined with state/territory information.

Reply:
These are mostly constructed in `pudl.metadata.enums` where there's `US_STATES`, `US_TERRITORIES`, `US_STATES_TERRITORIES`, `CANADA_PROVINCES_TERRITORIES`, `EPACEMS_STATES` etc. that are used in concert to construct `ENUM` constraints for various fields.

There's also the state/province/territory to approximate timezone mappings in `pudl.transform.eia.APPROXIMATE_TIMEZONES`

Having all of this information in a single static table stored as a DataFrame/CSV rather than several dictionaries in different places or split between a table and some random dangling dictionaries seems like it would be much cleaner.  I've created issue #1958 and assigned it to myself. I'll put it in `pudl.metadata.dfs.POLITICAL_SUBDIVISIONS`